ambassador/tests/ambassador_test.py

```diff
@@ -496,7 +496,10 @@ def normalize_gold(gold: dict) -> dict:
 @pytest.mark.parametrize("directory", MATCHES)
 @standard_setup
 def test_config(testname, dirpath, configdir):
-    global logger
+    pytest.xfail("old V1 tests are disabled")
```

Comment:
...well. Not yet. Does this break?

Reply:
Good point, I forgot why I did it at first place. I will look into this.it